content/docs/serverless-workflows/troubleshooting.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Troubleshooting
+date: "2024-07-25"
+weight: 101
+---
+
+
+# Troubleshooting Guide
+
+This document provides solutions to common problems encountered with serverless workflows.
+
+## Table of Contents
+
+1. [Workflow Errors](#workflow-errors)
+2. [Configuration Problems](#configuration-problems)
+3. [Performance Issues](#performance-issues)
+4. [Error Messages](#error-messages)
+5. [Network Problems](#network-problems)
+6. [Common Scenarios](#common-scenarios)
+7. [Contact Support](#contact-support)
+
+---
+
+## Workflow Errors
+
+### Problem: Workflow execution fails
+
+**Solution:**
+
+1. Examine the container log of the workflow
+    ```console
+        kubectl logs my-workflow-xy73lj
+    ```
+
+### Problem: Workflow is not listed by the orchestrator plugin
+
+**Solution:**
+
+1. Examine the container log of the workflow
+    ```console
+        kubectl logs my-workflow-xy73lj
+    ```
+
+2. Most probably the Data index service was unready when the workflow started.
```

Comment:
Should we include a sentence such as: *Workflows are being listed from the Data-Index service. Each workflow registers itself to the Data-Index service.*
Just to provide more context to this bullter.

Reply:
what about this comment? needed or not?